FND/Places/NorthAmerica/USPostalServiceAddressesIndividuals.rdf

```diff
@@ -0,0 +1,2119 @@
+		<rdfs:label>mews</rdfs:label>
+		<fibo-fnd-utl-av:commonDesignation>MEWS</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>MEWS</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Mill">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;StreetSuffix"/>
+		<rdfs:label>mill</rdfs:label>
+		<fibo-fnd-utl-av:commonDesignation>MILL</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>ML</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Mills">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;StreetSuffix"/>
+		<rdfs:label>mills</rdfs:label>
+		<fibo-fnd-utl-av:commonDesignation>MILLS</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>MLS</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Mission">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;StreetSuffix"/>
+		<rdfs:label>mission</rdfs:label>
+		<fibo-fnd-utl-av:commonDesignation>MISSN</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>MSSN</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>MSN</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Motorway">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;StreetSuffix"/>
+		<rdfs:label>motorway</rdfs:label>
+		<fibo-fnd-utl-av:commonDesignation>MOTORWAY</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>MTWY</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Mount">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;StreetSuffix"/>
+		<rdfs:label>mount</rdfs:label>
+		<fibo-fnd-utl-av:commonDesignation>MNT</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>MOUNT</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>MT</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>MT</fibo-fnd-utl-av:preferredDesignation>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-fnd-plc-uspsai;Mountain">
+		<rdf:type rdf:resource="&fibo-fnd-plc-adr;StreetSuffix"/>
+		<rdfs:label>mountain</rdfs:label>
+		<fibo-fnd-utl-av:commonDesignation>MNTAIN</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>MNTN</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>MOUNTAIN</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>MOUNTIN</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>MTIN</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:commonDesignation>MTN</fibo-fnd-utl-av:commonDesignation>
+		<fibo-fnd-utl-av:preferredDesignation>MTN</fibo-fnd-utl-av:preferredDesignation>
+	</owl:Named
```

Comment:
No, it's not a region, See earlier comment

Reply:
Not sure what you are referencing here - a street suffix isn't a region.